tutorials/asr/ASR_TTS_Tutorial.ipynb

```diff
@@ -0,0 +1,797 @@
+    "    model.tts_model_path=<path to compatible TTS model> \\\n",
+    "    model.enhancer_model_path=<optional path to enhancer model> \\\n",
+    "    model.asr_model_fuse_bn=<true recommended if ConformerEncoder with BatchNorm, false otherwise> \\\n",
+    "    model.train_ds.manifest_filepath=<path to manifest with audio-text pairs or null> \\\n",
+    "    model.train_ds.text_data.manifest_filepath=<path(s) to manifest with train text> \\\n",
+    "    model.train_ds.text_data.speakers_filepath=<path(s) to speakers list> \\\n",
+    "    model.train_ds.text_data.tokenizer_workers=4 \\\n",
+    "    model.validation_ds.manifest_filepath=<path to validation manifest> \\\n",
+    "    model.train_ds.batch_size=<batch_size>\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b17c097-a3b1-49a3-8f54-f07b94218d0b",
+   "metadata": {},
+   "source": [
+    "### Training a New Model from Scratch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d75b928-57b3-4180-bd09-37e018eef7ef",
+   "metadata": {},
+   "source": [
+    "```shell\n",
+    "python speech_to_text_bpe_with_text.py \\\n",
+    "    # (Optional: --config-path=<path to dir of configs> --config-name=<name of config without .yaml>) \\\n",
+    "    ++asr_model_type=<rnnt_bpe, ctc_bpe or hybrid_rnnt_ctc_bpe> \\\n",
+    "    ++tts_model_path=<path to compatible tts model> \\\n",
+    "    ++enhancer_model_path=<optional path to enhancer model> \\\n",
+    "    model.tokenizer.dir=<path to tokenizer> \\\n",
+    "    model.tokenizer.type=\"bpe\" \\\n",
+    "    model.train_ds.manifest_filepath=<path(s) to manifest with audio-text pairs or null> \\\n",
+    "    ++model.train_ds.text_data.manifest_filepath=<path(s) to manifests with train text> \\\n",
+    "    ++model.train_ds.text_data.speakers_filepath=<path(s) to speakers list> \\\n",
+    "    ++model.train_ds.text_data.min_words=1 \\\n",
+    "    ++model.train_ds.text_data.max_words=45 \\\n",
+    "    ++model.train_ds.text_data.tokenizer_workers=4 \\\n",
+    "    model.validation_ds.manifest_filepath=<path(s) to val/test manifest> \\\n",
+    "    model.train_ds.batch_size=<batch size> \\\n",
+    "    trainer.max_epochs=<num epochs> \\\n",
+    "    trainer.num_nodes=<number of nodes> \\\n",
+    "    trainer.accumulate_grad_batches=<grad accumultion> \\\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9a9a6cd3-4bdc-4b6e-b4b1-3bfd50fd01b3",
+   "metadata": {},
+   "source": [
+    "## Conclusion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2890c61-e4b7-47aa-a086-bc483ae7141f",
+   "metadata": {},
+   "source": [
+    "The tutorial demonstrated the main concepts related to hybrid ASR-TTS models to finetune ASR models and train new ones from scratch. \n",
+    "The ability to achieve good text-only adaptation results is demonstrated by finetuning a small Conformer model on text-only data from the AN4 dataset."
```

Comment:
I believe the Conclusion section is optional. You can leave it though if you like it. Or you can make it "Further Reading" or something.

Reply:
I think it's nice to have this section. I'd suggest renaming this to "Summary"

Reply:
Renamed to "Summary"